components/clp-package-utils/clp_package_utils/general.py

```
@@ -132,59 +130,59 @@ def generate_container_name(job_type: str) -> str:
-def check_dependencies():
+def is_docker_compose_running(project_name: str) -> bool:
+    """
+    Checks if a Docker Compose project is running.
+
+    :param project_name:
+    :return: Whether at least one instance is running.
+    :raises EnvironmentError: If Docker Compose is not installed or fails.
+    """
+    cmd = ["docker", "compose", "ls", "--format", "json", "--filter", f"name={project_name}"]
+    try:
+        output = subprocess.check_output(cmd, stderr=subprocess.STDOUT)
```

Ruff (0.14.0)

`subprocess` call: check for execution of untrusted input

(S603)